components/BacktoTopSection.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react'
+import Scroll from 'react-scroll'
+
+export default class TopArrowSection extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      atTopOfPage: true
+    }
+  }
+
+  componentDidMount () {
+    window.addEventListener('scroll', this.handleScroll)
```

Comment:
This handler will not be called when the page is first loaded. So we also need to call it manually on `componentDidMount`:

```js
    this.handleScroll()
```